graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH6.kt

```diff
@@ -0,0 +1,100 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphFunctions
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+@BEH(GraphBehaviours.HasCycles, GraphFunctions.HasCycles, """
+    Determine if a given directed graph contains one or more cycles
```

Comment:
🚫 Since we decided to use a single behaviour for both directed and undirected graphs, the behaviour description should drop the mention of directed graphs. It should be included only in the specific ACs that require it.
```suggestion
    Determine if a given graph contains one or more cycles
```